shoop/admin/static_src/product/edit_media.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * This file is part of Shoop.
+ *
+ * Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+$(function() {
+    $(".product-media-delete").on("click", function (e) {
+        e.preventDefault();
+        if(confirm("Are you sure?"))
+        {
+            $(this).parents(".panel").fadeOut();
+            $(this).next(".hide").find("input").attr("checked", true);
+        }
+    });
+
+    $(document).on("click", ".set-as-primary", function(e){
+        e.preventDefault();
+        var $panel = $(this).parents(".panel");
+        var prefix = $panel.data("prefix");
+        var pieces = prefix.split("-");
```

Comment:
Enable `"es6": true` in the Gulp configuration for this file, and you can use destructuring:

```javascript
var [start, current] = prefix.split("-");
```